.github/workflows/bump_version.yml

```diff
@@ -1,30 +1,40 @@
-name: Update Python Poetry packages
+name: bump up patch version using poetry
```

Comment:
Bump with a capital?
Change "bump up patch" to "bump patch" Bump up is a pleonasme